src/codegen.cpp

```diff
@@ -6432,6 +6432,7 @@ extern "C" void jl_init_codegen(void)
 #endif
 
 #ifdef USE_POLLY
+    cl::ParseEnvironmentOptions("Julia", "JULIA_POLLY_ARGS");
```

Comment:
Supplying arguments to Polly is not actually what the variable does, which makes this PR not ok to merge. Would the JULIA_LLVM_ARGS variable not be sufficient for what you need?